src/components/Main/Main.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[];
+  onDelete: (todoId: number) => void;
+  todosIdToDelete: number[];
+};
+
+export const Main: React.FC<Props> = ({ todos, onDelete, todosIdToDelete }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {todos.map(todo => {
+        const isTodoOnDelete = Boolean(
+          todosIdToDelete.find(id => todo.id === id),
+        );
```

Comment:
```suggestion
        const isTodoOnDelete = todosIdToDelete.includes(todo.id);
```
You can do it with `includes`